packages/opentelemetry-instrumentation-http/test/functionals/http-enable.test.ts

```diff
@@ -479,12 +479,29 @@ describe('HttpInstrumentation', () => {
         });
       }
 
-      it('should have 1 ended span when request throw on bad "options" object', () => {
+      it('should have 1 ended span when request throw on bad "options" object', done => {
         try {
-          http.request({ protocol: 'telnet' });
+          http.request({ headers: { cookie: undefined} }); // <===== this makes http.request throw
+          done('exception should have being thrown but did not');
         } catch (error) {
           const spans = memoryExporter.getFinishedSpans();
           assert.strictEqual(spans.length, 1);
+
+          const validations = {
+            httpStatusCode: undefined,
+            httpMethod: 'GET',
+            resHeaders: {},
+            hostname: 'localhost',
+            pathname: '/',
+            forceStatus: {
+              code: SpanStatusCode.ERROR, 
+              message: 'Invalid value "undefined" for header "cookie"',
```

Comment:
Looks like for some reason the message is different in node 8

Reply:
Yeah just noticed it. Do you think it's ok to check the node version in the test and assert the message accordingly?

Reply:
yes that's fine

Reply:
fixed. tests are green